src/main/frontend/components/block.cljs

```diff
@@ -275,16 +275,18 @@
       (lightbox/preview-images! images))))
 
 (defonce *resizing-image? (atom false))
-(rum/defcs resizable-image <
+(rum/defcs ^:large-vars/data-var resizable-image <
```

Comment:
:large-vars/cleanup-todo would be more accurate since data-var is used for vars that have to be long because of their data e.g. https://github.com/logseq/logseq/blob/040deb7212ebca262380c045af7a7440a80a35ca/src/main/frontend/dicts.cljc#L5. I can update this